pkg/router/router.go

```diff
@@ -634,3 +659,72 @@ func (r *router) IntroduceRules(rules routing.EdgeRules) error {
 		}
 	}
 }
+
+// RoutesCount returns count of the routes stored within the routing table.
+func (r *router) RoutesCount() int {
```

Comment:
Would it be easier just to have a function: `(*router).Table() routing.Table` to reduce clutter?

Reply:
@evanlinjin well, I just proxied all calls to the table through router, there's outer call from visor to get this count, so I left it as is to ensure compatibility